examples/lines_bars_and_markers/masked_demo.py

```diff
@@ -1,30 +1,46 @@
-'''
-===========
-Masked Demo
-===========
+"""
+==============================
+Plotting masked and NaN values
+==============================
 
-Plot lines with points masked out.
+Sometimes you need to plot data with missing values.
 
-This would typically be used with gappy data, to
-break the line at the data gaps.
-'''
+One possibility is to remove the undesired points from the data. In this case,
+a plotted line throughout the data is contignuous. It connects all given data
+points.
+
+Alternatively, the undesired points can be removed either using a
```

Comment:
Even if you don’t take my suggestion above, the “either” here should be removed or the second clause of the sentence constructed...